### PR TITLE
Calendar task

### PR DESCRIPTION
**Pull request by mttony**
- [DEMO LINK](https://mttony.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+    }
+  }
+
+  &__day {
+    @include square($cell-size);
+    background-color: $cell-color;
+    border: $border solid $border-color;
+    text-align: center;
+    line-height: 100px;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (TarasHoliuk) [inline src/styles/main.scss]: It's better to set `font-family` in `html` selector

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+    font-size: 30px;
+    transition-duration: 0.5s;
```
Comment (TarasHoliuk) [inline src/styles/main.scss]: set `transition-property` as well

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+
+    &:hover {
+      cursor: pointer;
```
Comment (TarasHoliuk) [inline src/styles/main.scss]: move it to `&__day`

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,59 @@
+@import "./utils/variables";
+@import "./utils/mixins";
```
Comment (TarasHoliuk) [inline src/styles/main.scss:2]: you haven't added it, so I can't check it on GH

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,59 @@
...
   margin: 0;
 }
+
+.c-container {
+  display: flex;
+  justify-content: center;
+  padding-top: 70px;
+}
+
+.calendar {
```
Comment (TarasHoliuk) [inline src/styles/main.scss]: according to BEM, all blocks should have it's own files (`container` too)

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ! But let's add a few improvements

File: src/styles/blocks/container.scss [hunk 6]
```diff
@@ -0,0 +1,5 @@
+.c-container {
```
Comment (loralevitska) [inline src/styles/blocks/container.scss]: Why `c-container`? 
```suggestion
.container {
```

File: src/styles/utils/_mixins.scss [hunk 7]
```diff
@@ -0,0 +1,4 @@
+@mixin square($size) {
+  height: $cell-size;
+  width: $cell-size;
+}
```
Comment (loralevitska) [inline src/styles/utils/_mixins.scss]: You have `$size` in params but use `$cell-size`. Remove redundant values

**Review by loralevitska — APPROVED** [review]
Good job!

**Pull request closed without merging**